frontend/src/components/DeleteVideo.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from "react";
+
+function DeleteVideo() {
+  const [videos, setVideos] = useState([]);
+  const [selectedVideos, setSelectedVideos] = useState([]);
+  const [message, setMessage] = useState("");
+
+  const fetchVideos = async () => {
+    try {
+      // Code pour récupérer toutes les vidéos de la base de données
+      const response = await fetch("http://localhost:5002/videos"); // Remplacez "/api/videos" par l'URL de votre endpoint API pour récupérer les vidéos
```

Comment:
Use a `.env` file instead of this comment. In addition, it is not "/api/videos" that you need to custom but "http://localhost:5002"